apps/rush-lib/src/data/RushConfiguration.ts

```diff
@@ -320,6 +321,13 @@ export default class RushConfiguration {
           + ` ${commonRushConfigFolder}`);
       }
     }
+
+    const pinnedVersionsFilename: string = path.join(commonRushConfigFolder, RushConstants.pinnedVersionsFilename);
+    if (fsx.existsSync(pinnedVersionsFilename)) {
+      throw new Error('The "pinned-versions.json" config file is no longer supported;'
```

Comment:
>Error [](start = 16, length = 5)

We should probably link to a wiki page describing how to migrate.

Reply:
I added a link to the web site.  They can read the "common-versions.json" config file docs.

---
In reply to: [178635948](https://github.com/Microsoft/web-build-tools/pull/593#discussion_r178635948) [](ancestors = 178635948)